src/pages/Pokemon/Evolution.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Image from "./Image";
+
+const Evolution = () => {
+
+  return (<div>
```

Comment:
mate I'd say no need to wrap the image `div` but since this is just a placeholder component it doesn't matter in this case